about.html

```diff
@@ -0,0 +1,150 @@
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+    crossorigin="anonymous"></script>
+	<link rel="stylesheet" href="style.css">
+	<script defer src="script.js"></script>
+  <title>About Us</title>
+</head>
+
+<body>
+  <!-- Header & Navigation -->
+  <header>
+	<!-- header-1 -->
+    <div class="first-header">
+      <ul class="first-header-ul">
+        <li><a><img src="Img/facebook-letter-logo-svgrepo-com.svg" alt="Facebook"></a></li>
+        <li><a><img src="Img/twitter-white.svg" alt="twitter"></a></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+	<!-- header-2 -->
+    </div>
+    <div class="nav-div">
+      <div class="logo">
+					<a href="index.html"
+						><img class=logo src="Img/daankwaLogo.png"  alt="Logo"
+					/></a>
+				</div>
+				<ul class="nav justify-content-end">
+					<li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="about.html">About</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="index.html#program">Program</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="./index.html#join">Join</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="./index.html#sponsors">Sponsor</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">News</a>
+        </li>
+        <li class="nav-item campaign">
+          <a class="nav-link" href="#">Talents</a>
+        </li>
+      </ul>
+      <div class="hamburger">
+					<img
+						src="Img/menu-sharp-custom.svg"
+						alt="Menu"
+					/>
+				</div>
+			</div>
+		</header>
+		<!-- Content Body -->
+		<div class="main">
+			<!-- Section Headline -->
+			<section class="headline">
+				<div class="wrapper into">
+					<h1>"When it comes to forever living Daa Nkwa is the choice"</h1>
+					<h2>Corporater Run</h2>
+									<p class="celebration">
+						The forever living project focuses on sustainable farming practices to enhance crop production and promote environmental stewardship. The project aims to implement innovative techniques such as organic farming, precision agriculture, and water conservation methods. By utilizing organic fertilizers, minimizing pesticide use, and adopting efficient irrigation systems, the project seeks to improve soil health, optimize resource utilization, and reduce the ecological footprint of farming operations. <h6>
+						Please contact us via email for further information and clarifications
```

Comment:
- [ ]  I can see you've center the content but again I think that you not only need to center the content but also to text-align your text into the center, as it is designed from behance design.  The text background used is also different from what was recommended. According to the design and the previous reviewer recommendation, you should use white background color.